File: week-1/cash/cash.c
#include <stdio.h>

int main()
{
    int troco = -1,moedas = 0;
    printf("Change owed: ");
    do
    {
        setbuf(stdin, NULL);
        scanf("%d", &troco);
    } while (troco < 0);
    for (; troco >= 25; troco -= 25)
    {
        moedas++;
    }
    for (; troco >= 10; troco -= 10)
    {
        moedas++;
    }
    for (; troco >= 5; troco -= 5)
    {
        moedas++;
    }
    for (; troco >= 1; troco -= 1)
    {
        moedas++;
    }
    printf("%d\n", moedas);
    return 0;
}

File: week-6/cash/cash.py
troco = -1
moedas = 0
while(troco < 0):
    troco = int(input("Change owed:"))
while(troco>=25):
    troco -= 25
    moedas+=1
while(troco>=10):
    troco -= 10
    moedas+=1
while(troco>=5):
    troco -= 5
    moedas+=1
while(troco>=1):
    troco -= 1
    moedas+=1
print(moedas)